modules/custom_operations/user_ie_extensions/sparse_conv_transpose.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "sparse_conv_transpose.hpp"
+
+using namespace TemplateExtension;
+
+//! [op:ctor]
+SparseConvTranspose::SparseConvTranspose(const ov::Output<ov::Node>& features,
+                                         const ov::Output<ov::Node>& inp_pos,
+                                         const ov::Output<ov::Node>& out_pos,
+                                         const ov::Output<ov::Node>& kernel,
+                                         const ov::Output<ov::Node>& offset) : Op({features, inp_pos, out_pos, kernel, offset}) {
+    constructor_validate_and_infer_types();
+}
+//! [op:ctor]
+
+//! [op:validate]
+void SparseConvTranspose::validate_and_infer_types() {
+    auto outShape = get_input_partial_shape(2);
+    auto kernelShape = get_input_partial_shape(3);
+    outShape[1] = kernelShape[4];
+    set_output_type(0, get_input_element_type(0), outShape);
+}
+//! [op:validate]
+
+//! [op:copy]
+std::shared_ptr<ov::Node> SparseConvTranspose::clone_with_new_inputs(const ov::OutputVector& new_args) const {
+    OPENVINO_ASSERT(new_args.size() == 5, "Incorrect number of new arguments");
+    return std::make_shared<SparseConvTranspose>(new_args.at(0), new_args.at(1), new_args.at(2), new_args.at(3), new_args.at(4));
+}
+//! [op:copy]
+
+//! [op:visit_attributes]
+bool SparseConvTranspose::visit_attributes(ov::AttributeVisitor& visitor) {
+    return true;
+}
+//! [op:visit_attributes]
```

Comment:
@ilyachur, we shouldn't require to implement this method if there are no attributes in an op. It makes more boilerplate code without any value. You can see, all ops in this PR don't have any attributes -- this is most likely true for all PT custom ops.

Reply:
Yes this behaviour was changed for the master branch, you can remove empty `visit_attributes` methods.
```suggestion
```